AnselemOdims/iReporter
# 162497998 chore UI and endpoints improvement

### What does this PR do?
Improves the UI and updates the endpoints

#### Description of Task to be completed?
- Include view detail button in dashboard
- Reduce the font size
- Work on application responsiveness
- Align the profile page to be at the center
- Align the reports records header in the dashboard to be at the center
- On active, text colour should be stable
- Create view report details page
- Make boxes display inline block
- The status button should align center
- Add dummy pagination buttons
- Improve the view reports page design
- Design the edit reports page
- Refactor code to fix bug and clear heroku error
- Refactor code to pass test
#### How can it be manually tested
- Clone this repository
- cd into the directory
- Do git checkout chore-ui-and-endpoints-improvement-162497998
- Check UI improvements on browser
- Host and deploy site to heroku
- Also, test the endpoints with postman
